bootstrap.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
-kubectl apply -f .infrastructure/mysql/ns.yml
-kubectl apply -f .infrastructure/mysql/configMap.yml
-kubectl apply -f .infrastructure/mysql/secret.yml
-kubectl apply -f .infrastructure/mysql/service.yml
-kubectl apply -f .infrastructure/mysql/statefulSet.yml
+# kubectl apply -f .infrastructure/mysql/ns.yml
+# kubectl apply -f .infrastructure/mysql/configMap.yml
+# kubectl apply -f .infrastructure/mysql/secret.yml
+# kubectl apply -f .infrastructure/mysql/service.yml
+# kubectl apply -f .infrastructure/mysql/statefulSet.yml
 
-kubectl apply -f .infrastructure/app/ns.yml
+# kubectl apply -f .infrastructure/app/ns.yml
 kubectl apply -f .infrastructure/app/pv.yml
 kubectl apply -f .infrastructure/app/pvc.yml
 kubectl apply -f .infrastructure/app/secret.yml
 kubectl apply -f .infrastructure/app/configMap.yml
 kubectl apply -f .infrastructure/app/clusterIp.yml
-kubectl apply -f .infrastructure/app/nodeport.yml
+# kubectl apply -f .infrastructure/app/nodeport.yml
```

Comment:
The NodePort service is commented out. If external access to the application is needed, ensure that this is configured appropriately elsewhere.